bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -335,11 +342,6 @@ while numpy.any(fnlouder == 0):
             f['foreground/ifar'][orig_fore_idx] = sec_to_year(ifar[i])
             f['foreground/fap'][orig_fore_idx] = fap[i]
 
-# If there were hierarchically removed foreground triggers, write the indices
-# of hierarchically removed foreground triggers to file
-if h_iterations > 0:
-     f['hierarchical_removed_fore_indices'] = save_these_fore_ind
-
```

Comment:
I don't know why it's saying all of these things here, this isn't being removed..... Maybe somebody with better git understanding can explain.